src/server/pps/persist/persist.proto

```diff
@@ -52,6 +52,9 @@ message PipelineInfos {
   repeated PipelineInfo pipeline_info = 1;
 }
 
+message SubscribeNewPipelineRequest {
+}
```

Comment:
No need to define an empty request, you can just have the method take `google.protobuf.Empty` if there's no arguments needed like `ListPipelineInfos` does.